src/catalogue/items/catalogueItemsTable.component.tsx

```diff
@@ -772,6 +772,14 @@ const CatalogueItemsTable = (props: CatalogueItemsTableProps) => {
         </>
       );
     },
+    renderBottomToolbarCustomActions: ({ table }) => (
+      <Typography>
```

Comment:
Could the padding be adjusted to align when there is grouping (/to match the other side's with the page buttons)
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/d63a689f-f9ca-4755-8157-04500d092446)